### PR TITLE
feat: add `ts-protoc-gen` plugin

### PR DESCRIPTION
**Pull request by markin-io**
PR duplicates stalled https://github.com/rvolosatovs/docker-protobuf/pull/105 as requested by @strophy

### CODE REVIEW

**Comment by rvolosatovs** [comment]
Thanks for contribution, please fix CI

File: deps.list [hunk 1]
```diff
@@ -3,8 +3,7 @@ GO_VERSION=1.19.3
 GOOGLE_API_VERSION=9de27c9c549173eac194ec80153963234a05a8b5
 GRPC_GATEWAY_VERSION=2.14.0
 GRPC_RUST_VERSION=0.8.3
-GRPC_SWIFT_VERSION=1.13.0
```
Comment (rvolosatovs) [inline deps.list:6]: Why is this removed?

File: Dockerfile [hunk 2]
```diff
@@ -329,6 +343,7 @@ RUN find /out -name "*.a" -delete -or -name "*.la" -delete
 FROM alpine:${ALPINE_VERSION}
 LABEL maintainer="Roman Volosatovs <rvolosatovs@riseup.net>"
 COPY --from=upx /out/ /
+COPY --from=protoc_gen_ts /out/ /
```
Comment (rvolosatovs) [inline Dockerfile:346]: Why not compress with UPX like other plugins?

File: deps.list (hunk 1 above)
Comment (strophy) [reply deps.list:6]: My bad, made a mistake resolving conflicts :see_no_evil:

**Review by rvolosatovs — APPROVED** [review]

File: deps.list [hunk 3]
```diff
@@ -3,8 +3,6 @@ GO_VERSION=1.19.3
 GOOGLE_API_VERSION=9de27c9c549173eac194ec80153963234a05a8b5
```
Comment (rvolosatovs) [inline deps.list]: Can we drop this merge commit? (https://github.com/rvolosatovs/docker-protobuf/pull/113/commits/a52f42d95d32661b108b43bdbd05d64bfa3c81b4)

**Review by rvolosatovs — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (strophy) [reply Dockerfile:346]: Discussed with markin-io and agreed to compress with UPX, tested working locally

File: deps.list (hunk 3 above)
Comment (strophy) [reply deps.list]: Sorry about that! I'm a relative novice at git, so I'm accustomed to messing around on branches and using squash merges to make it look nice afterwards. I'd love to learn better git hygiene, we can definitely drop this if you do it (or show me how)

Comment (rvolosatovs) [reply deps.list]: Rebased on top of `origin/main` and squashed fixups into feature commit (e42e045)

**Comment by rvolosatovs** [comment]
@strophy looking at CI log, I am not sure UPX works for the TS plugin. We can just drop the commit, merge this PR and perhaps file an issue to investigate whether this is at all possible

**Review by rvolosatovs — APPROVED** [review]
